src/core/tests/type_prop/rms_norm.cpp

```diff
@@ -0,0 +1,285 @@
+
+TEST_F(TypePropRMSNormTest, default_ctor) {
+    const auto op = make_op();
+    const auto data = std::make_shared<Parameter>(element::f16, PartialShape{2, 3, 8, 6});
+    const auto axes = std::make_shared<Parameter>(element::i64, PartialShape{1});
+    const auto scale = std::make_shared<Parameter>(element::f16, PartialShape{});
+
+    op->set_arguments(ov::OutputVector{data, axes, scale});
+    op->validate_and_infer_types();
+
+    EXPECT_EQ(op->get_output_size(), 1);
+    EXPECT_EQ(op->get_input_size(), 3);
+    EXPECT_EQ(op->get_output_element_type(0), element::f16);
+    EXPECT_EQ(op->get_output_partial_shape(0), (PartialShape{2, 3, 8, 6}));
+}
+
+TEST_F(TypePropRMSNormTest, no_scale_no_compute_type) {
+    const auto data = std::make_shared<Parameter>(element::f32, PartialShape{2, 3, 8, 6});
+    const auto axes = std::make_shared<Parameter>(element::i32, PartialShape{1});
+    const auto eps = 1e-5;
+
+    const auto op = make_op(data, axes, eps);
+    EXPECT_EQ(op->get_input_size(), 2);
+    EXPECT_EQ(op->get_output_size(), 1);
+    EXPECT_EQ(op->get_output_element_type(0), element::f32);
+    EXPECT_EQ(op->get_output_partial_shape(0), (PartialShape{2, 3, 8, 6}));
+    EXPECT_EQ(op->get_epsilon(), eps);
+}
+
+TEST_F(TypePropRMSNormTest, scale_no_compute_type) {
+    const auto data = std::make_shared<Parameter>(element::f16, PartialShape{2, 3, 8, 6});
+    const auto axes = std::make_shared<Parameter>(element::i32, PartialShape{1});
+    const auto scale = std::make_shared<Parameter>(element::f16, PartialShape{});
+    const auto eps = 1e-5;
+
+    const auto op = make_op(data, axes, scale, eps);
+    EXPECT_EQ(op->get_input_size(), 3);
+    EXPECT_EQ(op->get_output_size(), 1);
+    EXPECT_EQ(op->get_output_element_type(0), element::f16);
+    EXPECT_EQ(op->get_output_partial_shape(0), (PartialShape{2, 3, 8, 6}));
+    EXPECT_EQ(op->get_epsilon(), eps);
+}
+
+TEST_F(TypePropRMSNormTest, scale_compute_type) {
+    const auto data = std::make_shared<Parameter>(element::f16, PartialShape{2, 3, 8, 6});
+    const auto axes = std::make_shared<Parameter>(element::i32, PartialShape{1});
+    const auto scale = std::make_shared<Parameter>(element::f16, PartialShape{});
+    const auto eps = 1e-5;
+    const auto compute_type = element::f32;
+
+    const auto op = make_op(data, axes, scale, eps, compute_type);
+    EXPECT_EQ(op->get_input_size(), 3);
+    EXPECT_EQ(op->get_output_size(), 1);
+    EXPECT_EQ(op->get_output_element_type(0), element::f16);
+    EXPECT_EQ(op->get_output_partial_shape(0), (PartialShape{2, 3, 8, 6}));
+    EXPECT_EQ(op->get_epsilon(), eps);
+    EXPECT_EQ(op->get_compute_type(), compute_type);
+}
+
+TEST_F(TypePropRMSNormTest, scale_compute_type_no_scale) {
+    const auto data = std::make_shared<Parameter>(element::f16, PartialShape{2, 3, 8, 6});
+    const auto axes = std::make_shared<Parameter>(element::i32, PartialShape{1});
+    const auto eps = 1e-5;
```

Comment:
Nit: If common in test and the `TEST_F` it can be defined as some kind of default eps for test in test fixture.

Reply:
Ok, done